status/projects.py

```diff
@@ -150,19 +158,28 @@ def project_summary_data(self, row):
                 row.value["contact"] = (
                     ord_det["owner"]["name"] + ": " + ord_det["owner"]["email"]
                 )
+                field_sources["contact"] = (
+                    "LIMS (researcher.email) and Order Portal, formatted by Genomics Status (backend)"
```

Comment:
I guess its one or the other? I don't think researcher.email is updated any longer

Reply:
Maybe you can suggest a better source explanation? I'm a bit uncertain of the logics myself actually

Reply:
Fixed.